FreeRTOS/FreeRTOS-Kernel
# Add config option for event groups and stream buffers

Description
-----------
This PR adds config option for event group and stream buffer functionality.

1. Set `configUSE_STREAM_BUFFERS = 1` to enable stream buffers.
2. Set `configUSE_EVENT_GROUPS = 1` to enable event groups.

Test Steps
-----------
MPU demo on STM32H743ZI builds and runs successfully.

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
